site/kubernetes/operator/configure-operator-defaults.md

```diff
@@ -0,0 +1,155 @@
+<!--
+Copyright (c) 2020-2021 VMware, Inc. or its affiliates.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Configuring Defaults for RabbitMQ Cluster Operator
+
+## <a id="overview" class="anchor" href="#overview">Overview</a>
+
+This guide covers how to modify the default configuration of the [RabbitMQ Cluster Kubernetes Operator](/kubernetes/operator/operator-overview.html) in a Kubernetes cluster.
+
+It is possible to set environment variables on the operator Deployment in order to influence how the operator configures created RabbitmqClusters.
+This is most commonly useful when configuring the operator to automatically use RabbitMQ container images stored in a private registry.
```

Comment:
A very minor suggestion: could we avoid words such as `possible` `commonly useful` from the statement here? I found them making the statement vague and less certain.